packages/cli/test/utils/simulation/LodestarValidatorProcess.ts

```diff
@@ -58,6 +58,9 @@ export const LodestarValidatorProcess: ValidatorConstructor = class LodestarVali
     });
     this.secretKeys = validatorSecretKeys;
 
+    // Split half of the keys to external signer
+    this.externalSigner = new ExternalSignerServer(this.secretKeys.slice(0, this.secretKeys.length / 2));
```

Comment:
can we define and set the ranges to be divided locally and externally as validables (and the default split as half/half)

Reply:
also `this.rcConfig["externalSigner.url"] = this.externalSigner.url;` isn't required for this using this external signer?

Reply:
>also this.rcConfig["externalSigner.url"] = this.externalSigner.url; isn't required for this using this external signer?

oh ok since we now start keymanager to use this external signer

Reply:
@g11tech Done.